src/components/JobOpeningAboutUs.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import { View, Text, Button } from "native-base";
+import { COLOR } from "../styles/color";
+import { ABOUT_US } from "../helper/constant";
+
+const ProfileView = props => {
+  return (
+    <View style={styles.descriptionView}>
+      <Text style={styles.textStyle}>About Excellence Technologies</Text>
+      <View style={styles.aboutUsView}>
+        <Text style={styles.aboutUs}>{ABOUT_US}</Text>
+      </View>
+      {/* <View style={{ alignContent: "center" }}>
```

Comment:
what is this

Reply:
sir it is a section in Job Openings page